### ISSUE TITLE
6.3 Standardization of validation and test data

### ISSUE THREAD

**Issue by gmohandas** [fchollet/deep-learning-with-python-notebooks#100]
I have 2 concerns:
1. The training data is standardized using the mean and standard deviation of the training set. Shouldn't the validation and test set be standardized using its respective mean and standard deviation as well? I can't see this done anywhere.
2. Wouldn't it be better to refer to the procedure as standardizing and not normalizing? If nothing else, this will conform to scikit-learns terminology.

**Comment by gmohandas** [comment]
I withdraw the issue as I understand now that it is standard practice to standardize or normalize the validation and test set with the mean and standard deviation of the training set. This is, so that, the same (inverse) transformations can be applied to the validation and test sets with the prediction results. My bad!
